addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -645,6 +643,8 @@ protected synchronized void addDeviceAsChannel(Device device) {
         // Default, depends on device type
 
         if (device != null) {
+            logger.debug("Add virtual device as channel: {}", device.getMetrics().getTitle());
```

Comment:
But in that no device is also added so the same logging does not apply, I can add a logging for the other case, but don't know what I should state there and if its a warning, most likely it will not occur otherwise we would have had some NPE's flying around for some